adops/setting-up-prebid-multiformat-in-dfp.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: Setting up Prebid Multiformat in DFP
+head_title: Setting up Prebid Multiformat in DFP
+description: Setting up Prebid Multiformat in DFP
+pid: 3
+hide: false
+top_nav_section: adops
+nav_section: tutorials
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Setting up Prebid Multiformat in DFP
+{: .no_toc}
+
+This page shows how to set up your ad server so that you can serve multiformat ads.
+
+Multiformat ads allow you to declare multiple media types on a single ad unit.  For example, you can set up one ad on the page that could show a banner, native, or outstream video ad, depending on which had the highest bid.
+
+{: .alert.alert-info :}
+For instructions, on how to set up multiformat ads from the engineering side, see [Show Multiformat Ads with Prebid.js]({{site.baseurl}}/dev-docs/show-multiformat-ads.html).
+
+* TOC
+{: toc }
+
+## Step 1. Add an Order
+
+In DFP, create a new order for each of your header bidding partners.
+
+Repeat this step and step 2 when you are adding a new partner.
+
+## Step 2. Add Line Items and Creatives for each Media Type
+
+Because of the way multiformat support works in Prebid.js, for each price bucket, you will need to set up line items for each of the following formats:
+
++ One for [banner and outstream video][bannerAdSetup].  Banners and outstream videos will serve into a DFP banner creative.
+
++ One for [native][nativeAdSetup].  Native ads will serve into a native creative with native format and styles
+
+### Banner/Outstream
+
+Follow the instructions for creating line items and creatives in [Send all bids to the ad server][bannerAdSetup], with the following changes:
+
++ Add key-value targeting for the `hb_format` key: `video, banner`
```

Comment:
The "video, banner" notation is a bit ambiguous.  Let's say:

Add key-value targeting for "hb_format" is ("banner" OR "video").

We should also include a screenshot here, since this is unique to multi-format support.

Reply:
Addressed in 418cab5 and 595002a